src/Player.ts

```diff
@@ -169,7 +169,9 @@ export class Player extends PhysicsEntity {
     private showHints = false;
 
     public playerConversation: PlayerConversation | null = null;
-    public speechBubble = new SpeechBubble(this.scene, this.x, this.y, "white", true);
+    public speechBubble = new SpeechBubble(
+        this.scene, this.x, this.y, undefined, undefined, undefined, undefined, true
```

Comment:
Bei der Anzahl an Argumenten würde man normalerweise schon auf ein Parameter-Objekt umschwenken, denke ich. Dann kann man sich auch diese ganzen `undefined` sparen, nur weil einer der hinteren props `true` sein muss.

Ist aber nicht kritisch, kann auch gerne erstmal so bleiben.

Reply:
Die ganze Applikation könnte imho Konzepte wie position, size, padding usw. gut gebrauchen, um überall die Parameterzahlen auf ein überschaubares Maß zu reduzieren. Auch was Typsicherheit angeht wäre damit nochmal was dazugewonnen. Mit den asymmetrischen Paddings wird das hier nämlich sogar noch heftiger. Sollten wir im Nachgang mal im größeren Stil angehen.

Reply:
> Das hier hattest du mal erwähnt, glaube ich...
> ![image](https://user-images.githubusercontent.com/46005619/89575006-5e925e00-d82d-11ea-9e2b-2041fef643d3.png)
> Liegt das vielleicht am Ausrufezeichen am Ende?

Die width-Berechnung kann hier noch nicht so richtig mit den variablen Breiten. Hatte ich bereits unter Known Issues. Gucke noch danach, von mir aus kann das auch noch in den PR mit rein, dann lasse ich ihn noch offen.

Reply:
> > Das hier hattest du mal erwähnt, glaube ich...
> > ![image](https://user-images.githubusercontent.com/46005619/89575006-5e925e00-d82d-11ea-9e2b-2041fef643d3.png)
> > Liegt das vielleicht am Ausrufezeichen am Ende?
> 
> Die width-Berechnung kann hier noch nicht so richtig mit den variablen Breiten. Hatte ich bereits unter Known Issues. Gucke noch danach, von mir aus kann das auch noch in den PR mit rein, dann lasse ich ihn noch offen.

Können wir gerne auch nach dem PR noch angehen. Ist aber auf jeden Fall was, was noch gefixt werden sollte. Das fällt leider an vielen Stellen im Laufe des Spiels auf.